src/app/new-project/page.tsx

```diff
@@ -44,26 +69,61 @@ const NewProject = () => {
               form={form}
               rules={{ required: val => !!val || 'שדה חובה' }}
             />
-            <InputField
-              id="email"
-              label="מייל"
-              form={form}
-              rules={{ required: val => !!val || 'שדה חובה' }}
-            />
-            <InputField
-              id="phone"
-              label="טלפון"
-              form={form}
-              rules={{ required: val => !!val || 'שדה חובה' }}
-            />
+            <div className="mt-6 grid grid-cols-2 gap-8">

```

Comment:
you should hook the checkboxes and radio boxes up to the form also just like the input below

Reply:
ok , i fixed it

Reply:
where is the fix?

Reply:
in this commit : [Connecting the check boxes and radio boxes to the form](https://github.com/LevadoTech/frumhire_ui/pull/16/commits/a034d695f22d6fcbe4877a6069167dbeb9affca3)